components/EditEventForm.js

```diff
@@ -80,15 +80,30 @@ class EditEventForm extends React.Component {
         event['endsAt'] = value;
       }
     }
+    let slugChanged = this.state.slugChanged;
+    if (fieldname === 'slug') {
+      slugChanged = true;
+    }
+    if (!slugChanged && fieldname === 'name') {
+      event['slug'] = event['name'].length
+        ? `${slugify(event['name'])}-${this.state.event.parentCollective.id}ev`.toLowerCase()
+        : `{eventName}-${this.state.event.parentCollective.id}ev`.toLowerCase();
```

Comment:
Not sure about this line, having `{eventName}-4222ev` displayed as a slug will not look nice. I'd just remove the first part.

```suggestion
        : `-${this.state.event.parentCollective.id}ev`;
```